docbot/action_test.go

```diff
@@ -0,0 +1,320 @@
+package main
+
+import (
```

Comment:
I suggest use [testify](https://github.com/stretchr/testify) to simplify assertions.

Reply:
Yes, it is simple. To reduce external dependencies, I did not use this.

Reply:
Make sense. IIRC Golang will download test dependencies even in production env :(